Assets/MRTK/Core/Inspectors/Icons/MRTK_Toolbox_AppBar.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
```

Comment:
I don't think we should be adding all these images to the core. It maybe feels like an SDK feature, especially since the prefabs it references exist there.